packages/universal-test-runner/src/index.ts

```diff
@@ -1,4 +1,6 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-// TODO: Node.js API
+export { run } from '../src/run'
+export { loadAdapter, builtInAdapters } from '../src/loadAdapter'
+export { readProtocol, ProtocolResult } from '../src/readProtocol'
```

Comment:
As mentioned in the main thread, can we please move ProtocolResult to the `@aws/universal-test-runner-types` package? You'll be able to depend on that one as well as `@aws/universal-test-runner` -- even though it's currently marked in the README as being a "private" package, third-party adapter authors will eventually have to depend on it, so we plan on keeping that package mostly as-is.